aperturedb/BlobNewestDataCSV.py

```diff
@@ -80,7 +80,7 @@ class BlobNewestDataCSV(CSVParser.CSVParser):
     ``` python
 
         data = ImageForceNewestDataCSV("/path/to/WorkingImageDataset.csv")
-        loader = ParallelLoader(db)
+        loader = ParallelLoader(client)
```

Comment:
I'm worried `client` can be a confusing name as well. Do we know what the standard naming is that database libraries in python or in general use?

I feel like `conn` or `connection` is what I've seen a lot of.

Reply:
Let's stay on after standup and decide upon this.

Reply:
So I brooded over the "memorable name" way of calling it, and I think it's not useful at this time. The vendors who do it are the big and popular companies, where the name of the product is synonymous with the brand. It'd be a good state to get to but I am focussed on keeping things simple to follow for now.